.github/workflows/ci.yml

```diff
@@ -48,10 +49,15 @@ jobs:
             --disable-maintainer-mode \
             --disable-valgrind \
             --with-oniguruma=builtin \
-            ${{ matrix.configure_flag }}
+            --host=${{ matrix.arch }}-linux-gnu \
+            --enable-static \
+            --enable-all-static
           make
-          strip jq
+          ${ARCH}-linux-gnu-strip jq
+          file ./jq
```

Comment:
Accidentally left in? i think it could make sense to keep

Reply:
It's debugging, but it doesn't hurt.

Reply:
The `file` line is to ensure the cross-compiled binary is what we expect. I guess it could be removed but it doesn't hurt to have that visibility.